### PR TITLE
Update Get-ADUser.md

### PR DESCRIPTION
**Pull request by YuriySamorodov**
Added another example for using quotes in Filter queries

### CODE REVIEW

File: docset/winserver2019-ps/activedirectory/Get-ADUser.md [hunk 1]
```diff
@@ -190,8 +190,13 @@ The following syntax uses Backus-Naur form to show how to use the PowerShell Exp
 
 For a list of supported types for \<value\>, type `Get-Help about_ActiveDirectory_ObjectModel`.
 
-Note: For String parameter type, PowerShell will cast the filter query to a string while processing the command. When using a string variable as a value in the filter component, make sure that it complies with the [PowerShell Quoting Rules](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_quoting_rules). For example, if the filter expression is double-quoted, the variable should be enclosed using single quotation marks:
-**Get-ADUser -Filter "Name -like '$UserName'"**. On the contrary, if curly braces are used to enclose the filter, the variable should not be quoted at all: **Get-ADUser -Filter {Name -like $UserName}**.
+Note: For String parameter type, PowerShell will cast the filter query to a string while processing the command. When using a string variable as a value in the filter component, make sure that it complies with the [PowerShell Quoting Rules](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_quoting_rules). For example, if the filter expression is double-quoted, the variable should be either
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Get-ADUser.md:193]:
```suggestion
Note: For String parameter type, PowerShell will cast the filter query to a string while processing the command. When using a string variable as a value in the filter component, make sure that it complies with the [PowerShell Quoting Rules](/powershell/module/microsoft.powershell.core/about/about_quoting_rules). For example, if the filter expression is double-quoted, the variable should be either:
```

File: docset/winserver2019-ps/activedirectory/Get-ADUser.md [hunk 2]
```diff
@@ -190,8 +190,13 @@ The following syntax uses Backus-Naur form to show how to use the PowerShell Exp
...
+enclosed using single quotation marks:
+**Get-ADUser -Filter "Name -like '$UserName'"**
+or two pairs of double quotes:
+**Get-ADUser -Filter "Name -like ""$UserName"""**
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Get-ADUser.md:197]:
```suggestion
- Enclosed using single quotation marks:
**Get-ADUser -Filter "Name -like '$UserName'"**
- Or enclosed using two pairs of double quotes:
**Get-ADUser -Filter "Name -like ""$UserName"""**
```

Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Get-ADUser.md:197]:
```suggestion
**Get-ADUser -Filter "Name -like ""$UserName"""**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @YuriySamorodov, kindly commit the minor suggested changes. Many thanks.

**Comment by YuriySamorodov** [comment]
Thanks @JohanFreelancer9 , this is completed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by e0i** [comment]
@tiburd This one can be merged. Thanks.

**Pull request merged**